src/create-glamorous.js

```diff
@@ -42,56 +42,19 @@ function createGlamorous(splitProps) {
        * This is a component which will render the comp (closure)
        * with the glamorous styles (closure). Forwards any valid
        * props to the underlying component.
-       * @param {Object} theme the theme object
-       * @return {ReactElement} React.createElement
        */
-      class GlamorousComponent extends Component {
```

Comment:
Able to remove quite a bit of code thanks to this change. Though we're now using the `withTheme` HoC so I'm pretty sure size will remain about the same.